_core/smart/tutorial-counter.md

```diff
@@ -169,6 +170,8 @@ Let's get familiar with the tests to understand what the new smart contract shou
         (ok (var-get counter))))
     ```
 
+---
+
 With the completion of this tutorial, you ...
 
 * Experienced test-driven development with Clarity
```

Comment:
why this, why do we need a tutorial for testing?

Reply:
We should be thinking about discoverability and SEO as we think about how to divvy up topics. Why should someone who needs to know about testing have to look in the sub-areas of the dev tutorials?

Reply:
I think testing is a default step in smart contract development. TDD seems to be the fastest development flow. either way, happy with any decision. anything is a step forward at this point ;)

Reply:
:) Let's keep this as is and see what the haxor research says.